json/src/value.rs

```diff
@@ -812,17 +813,181 @@ from_integer! {
 }
 
 impl From<f32> for Value {
+    /// Convert 32-bit floating point number to `Value`
```

Comment:
Yes, likes this:

![bildschirmfoto 2017-02-13 um 15 03 16](https://cloud.githubusercontent.com/assets/20063/22886320/91e82c68-f1fd-11e6-8e51-5cb74953fe9c.png)

(Also, the doc tests are nicer than unit tests here IMHO.)